tests/ClientTest.php

```diff
@@ -562,4 +563,13 @@ public function testGetSettlementsWithInvalidDateThrowsException()
         $this->expectException(ValidationException::class);
         $this->client->getSettlements('30.5.2022');
     }
+
+    public function testRequestPaymentReportReturnsOk()
```

Comment:
Do you think you would need to test that exceptions are thrown with invalid input?

Reply:
Fixed in #44